tests/test_geoprocessing.py

```diff
@@ -4186,3 +4186,40 @@ def test_convolve_2d_nodata(self):
         expected_output[0, 0] = 1
         expected_output[-1, -1] = 1
         numpy.testing.assert_allclose(target_array, expected_output)
+
+    def test_convolve_2d_gaussian(self):
+        """PGP.geoprocessing: test convolve 2d with large gaussian kernel."""
+        # choosing twice the max memory block
+        n_pixels = 256*2
+        # this is a fun seed random seed
+        numpy.random.seed(n_pixels)
+        signal_array = numpy.random.random((n_pixels, n_pixels))
+
+        workspace_dir = 'local'
+        if not os.path.exists(workspace_dir):
+            os.makedirs(workspace_dir)

```

Comment:
Should this be `self.workspace_dir`? so that it gets deleted on cleanup?

Reply:
Oh gosh of course it should, I do this a lot when I'm debugging tests and then I also forget to change it back. Changed in 3349d4b9eb4ba5d09fc8ba576e8fb13e00c8e657